packages/react/src/editor/styles.css

```diff
@@ -4,16 +4,21 @@
 
 /* Default theme params */
 .bn-container {
-    --bn-colors-editor-text: #3F3F3F;
-    --bn-colors-editor-background: #FFFFFF;
-    --bn-colors-menu-text: #3F3F3F;
-    --bn-colors-menu-background: #FFFFFF;
-    --bn-colors-tooltip-text: #3F3F3F;
+    /* TODO: Do we want this? If yes, we should probably have a similar
```

Comment:
comment for code below, should we prefix all mantine related stiles with .bn-container? (i.e.: `.mantine-Badge-root`)